cloudflare/resource_cloudflare_account_member.go

```diff
@@ -0,0 +1,114 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+// This needs to be hooked up to the provider schema instead.
+var cloudflareOrgID = os.Getenv("CLOUDFLARE_ORG_ID")
```

Comment:
Need to work out why `d.GetOk("org_id")` isn't working in the methods below.

Using `org_id` also changes a bunch of functionality to be managed since it overrides `user_owner_from_zone` which isn't used in the account member resource but changing any behaviour to include it may break other things.

https://github.com/terraform-providers/terraform-provider-cloudflare/blob/b9b9853d9b539375104f0a4eb22a1cd41b415fb9/cloudflare/provider.go#L73-L78